qutebrowser/browser/commands.py

```diff
@@ -1053,6 +1057,27 @@ def quickmark_load(self, name, tab=False, bg=False, window=False):
         url = objreg.get('quickmark-manager').get(name)
         self._open(url, tab, bg, window)
 
+    @cmdutils.register(instance='command-dispatcher', scope='window')
+    def bookmark_add(self):
+        """Save the current page as a bookmark."""
+        bookmark_manager = objreg.get('bookmark-manager')
+        bookmark_manager.add(self._win_id, self._current_url(),
+                             self._current_title())
+
+    @cmdutils.register(instance='command-dispatcher', scope='window',
+                       maxsplit=0,
+                       completion=[usertypes.Completion.bookmark_by_url])
+    def bookmark_load(self, url, tab=False, bg=False, window=False):
+        """Load a bookmark.
+
+        Args:
+            url: The url of the bookmark to load.
+            tab: Load the bookmark in a new tab.
+            bg: Load the bookmark in a new background tab.
+            window: Load the bookmark in a new window.
+        """
+        self._open(QUrl(url), tab, bg, window)
```

Comment:
If we want to be able to add bookmarks by hand, I believe `fuzzy_url` is the way to do. I find more convenient to add bookmarks by first navigating to the page and launching the `bookmark_add` command or, if I want to import a bunch of bookmarks, by copy-pasting in the urls file directly.